charts/jenkins/pod-templates/nuxeo-package-lts-2025.yaml.gotmpl

```
@@ -9,7 +9,7 @@
   {{- toYaml .Values.podTemplates.containers | nindent 2 -}}
   - args: ""
     command: ""
-    image: "{{ .Values.docker.registry }}/nuxeo/builder-java21-chrome-nodejs18:latest"
+    image: "{{ .Values.docker.registry }}/nuxeo/builder-java21:latest"
```

```suggestion
    image: "{{ .Values.docker.registry }}/nuxeo/builder-java21-chrome:latest"
```
I guess, since `nuxeo/builder-java21` doesn't have Chrome installed.